File: src/main/java/ru/zdoher/japs/repositories/sentence/SentencePolitenessRepository.java
package ru.zdoher.japs.repositories.sentence;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;
import ru.zdoher.japs.domain.sentence.SentencePoliteness;

@Repository
public interface SentencePolitenessRepository
        extends CrudRepository<SentencePoliteness, String> {
}

File: src/main/java/ru/zdoher/japs/repositories/sentence/SentencePolitenessRepository.kt
package ru.zdoher.japs.repositories.sentence

import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository
import ru.zdoher.japs.domain.sentence.SentencePoliteness

@Repository
interface SentencePolitenessRepository : ReactiveMongoRepository<SentencePoliteness, String>